src/com/magento/idea/magento2plugin/magento/packages/code/MagentoVersion.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+package com.magento.idea.magento2plugin.magento.packages.code;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public enum MagentoVersion {
+
+    ENTERPRISE_EDITION("magento/product-enterprise-edition", 1),
+    COMMUNITY_EDITION("magento/product-community-edition", 2);
```

Comment:
Are these the composer packages? If yes - how is supposed to work with projects installed via GitHub?

Reply:
@eduard13, it was not supported in the previous functional and is not supposed to be supported for now. We need correct version lookup for the UCT version auto-detection. You can create new issue for adding support of dev Magento version resolving, or I'll create issue for that letter.

Thanks, Bohdan